x/mint/keeper/keeper.go

```diff
@@ -286,29 +303,21 @@ func (k Keeper) distributeDeveloperRewards(ctx sdk.Context, totalMintedCoin sdk.
 				return sdk.Int{}, err
 			}
 			devRewardPortionCoins := sdk.NewCoins(devPortionCoin)
-			if w.Address == "" {
-				err := k.distrKeeper.FundCommunityPool(ctx, devRewardPortionCoins,
-					k.accountKeeper.GetModuleAddress(types.DeveloperVestingModuleAcctName))
-				if err != nil {
-					return sdk.Int{}, err
-				}
-			} else {
-				devRewardsAddr, err := sdk.AccAddressFromBech32(w.Address)
-				if err != nil {
-					return sdk.Int{}, err
-				}
-				// If recipient is vesting account, pay to account according to its vesting condition
-				err = k.bankKeeper.SendCoinsFromModuleToAccount(
-					ctx, types.DeveloperVestingModuleAcctName, devRewardsAddr, devRewardPortionCoins)
-				if err != nil {
-					return sdk.Int{}, err
-				}
+			devRewardsAddr, err := sdk.AccAddressFromBech32(w.Address)
```

Comment:
Removed `w.Address == ""` condition. The motivation is in the PR description. Diff is large due to tab changes

Reply:
Doesn't the block of code link in the PR description specifically allow `w.Address` to be an empty string? i.e. the block of code only runs if it's not empty, so if it is then the check that prevents empty strings in `sdk.AccAddressFromBech32` is bypassed?

Zoomed out block from what you linked in PR desc:
https://github.com/osmosis-labs/osmosis/blob/c0052bbdaebac0ec94085e7742184ae2141706b9/x/mint/types/params.go#L224-L230

Reply:
Oh you are right! Good catch! I totally misread that check and assumed the opposite. I think it's best to keep the original approach and add more tests for that behavior

Reply:
fixed